### PR TITLE
feat(bankAccounts): added support for verifying accounts

### PR DESCRIPTION
**Pull request by elnaz**
@lob/api-engineering-team 

https://github.com/lob/lob-python/issues/67

### CODE REVIEW

**Comment by leore** [comment]
build failing due to new verify. Need to use a verified bank account to send check .

**Comment by elnaz** [comment]
I fixed the unverified bank in the check tests. The automatic build failed but when I retried the build through Travis directly, it passed: https://travis-ci.org/lob/lob-python/builds/59281092

So this is ready for review @pon @leore

**Comment by pon** [comment]
LGTM

**Comment by leore** [comment]
Any idea why the buld is failing?

**Comment by pon** [comment]
Latest is passing: https://travis-ci.org/lob/lob-python/builds/59281092

File: examples/check.py [hunk 1]
```diff
@@ -47,7 +47,23 @@
 print "======================================================="
 print "\n"
 
-# Creating a Check using the previously created bank account
+# Verifying a Bank Account with the microdeposit amounts
```
Comment (leore) [inline examples/check.py]: what happens when you try to verify an account that has already been verified?

Comment (pon) [inline examples/check.py]: It errors

Comment (leore) [inline examples/check.py]: @pon @elnaz having the verify process in the check example might be weird if you cannot verify a bank over and over again

Comment (leore) [inline examples/check.py]: I dont think this will work then.

Comment (pon) [inline examples/check.py]: This will - the bank account is created above

Comment (leore) [inline examples/check.py]: Ahh yep.

**Comment by leore** [comment]
LGTM

**Comment by leore** [comment]
Let create a new version

**Pull request merged**